map.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable indent */
+const myMap = (arr, callback) => {
```

Comment:
This is a really nice, clean `map` function, and I love seeing the two types of for loops side by side.